### ISSUE TITLE
Link does not display if link title is not included

### ISSUE THREAD

**Issue by andybroomfield** [localgovdrupal/localgov_alert_banner#147]
Before if you did not include link text the link would use 'more information'
now the link does not display.
Was this feature removed? (see #10)

Steps to reproduce:
1. Create an alert banner, filling in only the url of the link field.
2. Set the banner live
3. On confirmation screen, and when the banner is live, the link is missing.

What should happen:
Link should be visible, with the label 'More information' or at least the banner link field, but really that should be avoided.

**Comment by andybroomfield** [comment]
Added a fix in the template by checking if the field renders.

We need to decide if 'More information' is a default thing that should get set, and how (either as a field default value or using preprocess as now or in a template).

**Comment by andybroomfield** [comment]
This is how the default link is set, in localgov_alert_banner.page.inc line 36.
````php
  // Set default link text if field exists.
  if ($entity->hasField('link')) {
    if ($entity->get('link')->title === '') {
      $variables['content']['link'] = Link::fromTextAndUrl(t('More information'), Url::fromUri($entity->get('link')->uri))->toString();
    }
  }
````
Doing it this way means that the whole link field is replaced by custom markup, so field tempalates, and I assume to `content.link.0 is not empty` twig condition will fail. The render fix in #148 fixes it, though maybe this is not the best implementation.
